### PR TITLE
WIP: Tpek backend

### PR DESCRIPTION
**Pull request by AntoineDupre**
implement backend bootsrap and backend storage for Sequester keys, see issue #2463 and #2464 and #2468

### CODE REVIEW

File: parsec/backend/postgresql/migrations/datamodel.sql [hunk 1]
```diff
@@ -25,9 +25,26 @@ CREATE TABLE organization (
...
+-------------------------------------------------------
+--  Sequester
+-------------------------------------------------------
 
+CREATE TYPE sequester_service AS ENUM ('SEQUESTRE', 'WEBHOOK');
+
+CREATE TABLE sequester(
+    _id SERIAL PRIMARY KEY,
+    service_type sequester_service NOT NULL,
+    service_id TEXT NOT NULL,
```
Comment (touilleMan) [inline parsec/backend/postgresql/migrations/datamodel.sql:40]: uuid ?

File: parsec/backend/postgresql/migrations/datamodel.sql [hunk 2]
```diff
@@ -25,9 +25,26 @@ CREATE TABLE organization (
...
     _bootstrapped_on TIMESTAMPTZ,
     _created_on TIMESTAMPTZ NOT NULL
+    sequester_verify_key BYTEA;
 );
 
+-------------------------------------------------------
+--  Sequester
+-------------------------------------------------------
 
+CREATE TYPE sequester_service AS ENUM ('SEQUESTRE', 'WEBHOOK');
```
Comment (touilleMan) [inline parsec/backend/postgresql/migrations/datamodel.sql:35]: `sequester_service_type` here and `sequester_service` for the table ?

File: parsec/backend/postgresql/migrations/datamodel.sql [hunk 3]
```diff
@@ -25,9 +25,26 @@ CREATE TABLE organization (
...
+
+CREATE TABLE sequester(
+    _id SERIAL PRIMARY KEY,
+    service_type sequester_service NOT NULL,
+    service_id TEXT NOT NULL,
+    organization INTEGER REFERENCES organization (_id) NOT NULL,
+    encryption_key_payload BYTEA,
+    encryption_key_payload_signature BYTEA,
+    webhook_url TEXT,
```
Comment (touilleMan) [inline parsec/backend/postgresql/migrations/datamodel.sql:44]: should add a comment to explain this is not NULL only if type is `WEBHOOK`

File: parsec/backend/postgresql/migrations/datamodel.sql [hunk 4]
```diff
@@ -25,9 +25,26 @@ CREATE TABLE organization (
...
+-------------------------------------------------------
 
+CREATE TYPE sequester_service AS ENUM ('SEQUESTRE', 'WEBHOOK');
+
+CREATE TABLE sequester(
+    _id SERIAL PRIMARY KEY,
+    service_type sequester_service NOT NULL,
+    service_id TEXT NOT NULL,
+    organization INTEGER REFERENCES organization (_id) NOT NULL,
+    encryption_key_payload BYTEA,
```
[the remark this answers is not in the document]
Comment (touilleMan) [reply parsec/backend/postgresql/migrations/datamodel.sql:42]: should be `NOT NULL`

File: parsec/backend/postgresql/migrations/datamodel.sql [hunk 5]
```diff
@@ -25,9 +25,26 @@ CREATE TABLE organization (
...
+    sequester_verify_key BYTEA;
 );
 
+-------------------------------------------------------
+--  Sequester
+-------------------------------------------------------
 
+CREATE TYPE sequester_service AS ENUM ('SEQUESTRE', 'WEBHOOK');
+
+CREATE TABLE sequester(
```
Comment (touilleMan) [inline parsec/backend/postgresql/migrations/datamodel.sql:37]: should add a fields:
```sql
    created_on TIMESTAMPTZ NOT NULL,
    -- NULL if not deleted
    deleted_on TIMESTAMPTZ,
```

**Comment by AntoineDupre** [comment]
Outdated, fixed by #2510

**Pull request closed without merging**